code-reviews/recipes/CSharp.md

```diff
@@ -1,22 +1,20 @@
 # C# Code Reviews
 
-## C# Style Guide
+## Style Guide
```

Comment:
Noticed this one doesn't have build validation and dependencies like the bash recipe. Is this intentional?

Reply:
I think @debugerr will add that as he updates the C# section.

Reply:
Absolutely.